### PR TITLE
Get options from options parameter :)

### PR DESCRIPTION
**Pull request by emattias**
**What**:
Makes options provided to `babel-preset-css-prop` be used.

**How**:
Takes the options from the second function argument where they are sent to the function :)

### CODE REVIEW

File: packages/babel-preset-css-prop/src/index.js [hunk 1]
```diff
@@ -8,14 +8,10 @@ let pragmaName = '___EmotionJSX'
...
-  pragma,
-  sourceMap,
-  autoLabel,
-  labelFormat,
-  instances,
-  ...options
-}) => {
+export default (
+  api,
+  { pragma, sourceMap, autoLabel, labelFormat, instances, ...options }
```
Comment (Andarist) [inline packages/babel-preset-css-prop/src/index.js]: is this guaranteed to always be an object?

Comment (emattias) [reply packages/babel-preset-css-prop/src/index.js]: `babel-preset-react` seems to assume that: https://github.com/babel/babel/blob/master/packages/babel-preset-react/src/index.js#L10 :)

Comment (emattias) [reply packages/babel-preset-css-prop/src/index.js]: Or actually they use this `declare` helper. Can we use that here also? 🤔 https://github.com/babel/babel/blob/master/packages/babel-helper-plugin-utils/src/index.js#L1

Comment (emattias) [reply packages/babel-preset-css-prop/src/index.js]: Or just default the argument to `= {}`?

**Comment by mitchellhamilton** [comment]
Could you add a test?

**Comment by mitchellhamilton** [comment]
Something like this:

```jsx
import { transform } from "@babel/core";
import preset from "@emotion/babel-preset-css-prop";
test("works with options", () => {
  transform(
    `
...some code
`,
    {
      presets: [[preset, { ...someOptions }]],
      babelrc: false,
      configFile: false
    }
  );
});
```

**Pull request merged**